README.md

```diff
@@ -420,6 +420,7 @@ python -c 'from django.core.management.utils import get_random_secret_key; \
 ```
 
 Please leave `SERVE_STATIC=False` because in the `docker-compose` configuration static files are served with nginx.
+If running on local you have to add localhost to allowed_hosts like this :`ALLOWED_HOSTS=localhost`.
```

Comment:
I'm not sure if this is needed. You should check docker-compose at `0.0.0.0` address. 

Maybe better that changing the readme will be to add a `localhost` at django settings in https://github.com/mljar/mercury/blob/d016d584117a061e175c5b9ac6c7ee3154442a03/mercury/server/settings.py#L46 (maybe add this as separate issue)